example/v2_framework.cpp

```diff
@@ -122,28 +125,27 @@ inline constexpr struct watt : named_unit<"W", joule / second> {
 
 namespace short_units {
```

Comment:
What do you think of the more formal name `unit_symbols`? The nested namespaces can keep the `_units` suffix.

Reply:
I like the proposed name :-)

Reply:
BTW, `_units` suffix was needed to not collide with `system_reference` name when the user will do something like: `using namespace units::isq::si;`. Anyway, I am not a fan of such a using directive as it makes `units::isq::si::time` collide with POSIX `time`.